### ISSUE TITLE
Include test case class for components

### ISSUE THREAD

**Issue by rainerborene** [github/actionview-component#109]
Here is what I'm currently using on my `test_helper.rb` file.

```ruby
class ActionView::ComponentTest < ActiveSupport::TestCase
  include ActionView::Component::TestHelpers
end
```

I think it's a good idea to provide this base for class for testing components.

**Comment by joelhawksley** [comment]
Sounds good to me! I’ll happily accept a PR ❤️

**Comment by kaspermeyer** [comment]
Resolved in #138
